htdocs/contrat/class/contrat.class.php

```diff
@@ -753,6 +753,7 @@ public function fetch($id, $ref = '', $ref_customer = '', $ref_supplier = '')
 			$this->error = $this->db->error();
 			return -1;
 		}
+		return -1;
```

Comment:
The only case we can reach this point is when we reach line 691: $result = -2
All other cases, the return is inside the if or else. So i suggest to remove this added line and instead make a return 0 line 691. 
So we match the doc comment saying that return is 0 when not found or when we found more than 1 record.